src/celengine/renderglsl.cpp

```diff
@@ -10,59 +10,114 @@
 // as published by the Free Software Foundation; either version 2
 // of the License, or (at your option) any later version.
 
-#include <config.h>
 #include <algorithm>
+#include <array>
+#include <cmath>
+#include <cstddef>
 #include <memory>
+
+#include <celmath/geomutil.h>
+#include <celmath/mathlib.h>
+#include <celmodel/material.h>
+#include <celutil/color.h>
+#include "body.h"
 #include "framebuffer.h"
 #include "geometry.h"
+#include "glsupport.h"
 #include "lodspheremesh.h"
-#include "meshmanager.h"
-#include "modelgeometry.h"
+#include "rendcontext.h"
 #include "render.h"
-#include "renderinfo.h"
 #include "renderglsl.h"
+#include "renderinfo.h"
 #include "shadermanager.h"
-#include "texmanager.h"
+#include "texture.h"
 #include "vecgl.h"
-#include <celengine/astro.h>
-#include <celmath/frustum.h>
-#include <celmath/distance.h>
-#include <celmath/geomutil.h>
-#include <celmath/intersect.h>
-#include <celutil/utf8.h>
-#include <celutil/util.h>
-#include "shadowmap.h"
-
-using namespace cmod;
-using namespace Eigen;
-using namespace std;
-using namespace celmath;
-using namespace celestia;
```

Comment:
please restore `using namespace celestia;` as all code will in celestia namespace so the future change will be minimal